<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxCategorie">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>350</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(228,224,223);
alternate-background-color: rgb(153, 204, 153);
font: 18pt &quot;Segoe UI&quot;;
border-radius:20%;</string>
    </property>
    <property name="title">
     <string>Produit</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:696;&quot;&gt;Idp &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:696;&quot;&gt;Nomp&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Catégorie &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:696;&quot;&gt;Marque&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:696;&quot;&gt;Prix&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:696;&quot;&gt;Quantité&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Idp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
 border-radius:10%;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="categorie">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
 border-radius:10%;</string>
     </property>
     <item>
      <property name="text">
       <string>Homme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Femme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enfant</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Nomp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
 border-radius:10%;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Modifier_produit">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>620</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Modifer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouter_produit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>620</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="expoter_produit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>620</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Exporter pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suppreime_produit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>620</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recherhce_produit">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>90</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
style:bold;
</string>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>-20</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Black_and_White_Elegant_Fashion_Brand_Logo-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:20%;
color: rgb(255, 255, 255);
font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>       Gestion des produits</string>
    </property>
   </widget>
   <widget class="QFrame" name="BarreMenu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>631</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>13</red>
          <green>33</green>
          <blue>79</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79);border-radius: 15px; 
QLabel {
    color: rgb(255, 255, 255); /* White text color for labels */
}</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>2</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>BarreMenu</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>86</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* White text color for labels */
}</string>
     </property>
     <property name="text">
      <string>👥Client</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>240</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* White text color for labels */
}</string>
     </property>
     <property name="text">
      <string>🛒Commande</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>340</y>
       <width>147</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* White text color for labels */
}</string>
     </property>
     <property name="text">
      <string>🏭Fournisseur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>144</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* White text color for labels */
}</string>
     </property>
     <property name="text">
      <string>🧾Produits</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>119</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* White text color for labels */
}</string>
     </property>
     <property name="text">
      <string>🚚Livraison</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* White text color for labels */
}</string>
     </property>
     <property name="text">
      <string>💼Employé</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="marque">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
 border-radius:10%;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrix">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
 border-radius:10%;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQuantite">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
 border-radius:10%;</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(228,224,223);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Idp_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
 border-radius:10%;
background-color:rgb(228,224,223);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trier_produit">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>620</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>trier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statistique_produit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>statistique</string>
    </property>
   </widget>
   <widget class="QPushButton" name="historique_produit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Historique</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>ardiuno</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
